src/Statistics/StatisticsProcessor.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Statistics;
+
+/**
+ * @internal
+ */
+interface StatisticsProcessor
```

Comment:
Subjective (sometimes unpopular) opinion (but consistent in the Symfony world): `StatisticsProcessorInterface`